sdk/enoki/src/EnokiClient/index.ts

```diff
@@ -90,6 +92,19 @@ export class EnokiClient {
 		});
 	}
 
+	executeSponsoredTransactionBlock(input: ExecuteSponsoredTransactionBlockApiInput) {
+		return this.#fetch<ExecuteSponsoredTransactionBlockApiResponse>(
+			`transaction-blocks/sponsor/${input.digest}`,
```

Comment:
I guess this was probably in another PR, but why is digest part of the URL here?

Reply:
just saw your slack message and found the other PR, mostly makes sense, I think I was thrown off by the bytes being sent in both messages

Reply:
Yeah tbh it doesn't _need_ to be, we could also store it in the DB, I'm just lazy. I think the digest makes sense as the key, but I'm open to iterate on the design here, I don't know what the most REST-ful way to do this would be.